index.js

```diff
@@ -124,33 +126,41 @@ class K8sExecutor extends Executor {
+     * @param  {String} [options.kubernetes.baseImage]                     Base image for the pod
+     * @param  {Number} [options.kubernetes.buildTimeout=90]               Number of minutes to allow a build to run before considering it is timed out
+     * @param  {Number} [options.kubernetes.maxBuildTimeout=120]           Max timeout user can configure up to
+     * @param  {String} [options.kubernetes.serviceAccount=default]        Service Account for builds
+     * @param  {String} [options.kubernetes.resources.cpu.max=12]          Upper bound for custom CPU value (in cores)
+     * @param  {String} [options.kubernetes.resources.cpu.turbo=12]        Value for TURBO CPU (in cores)
+     * @param  {String} [options.kubernetes.resources.cpu.high=6]          Value for HIGH CPU (in cores)
+     * @param  {Number} [options.kubernetes.resources.cpu.low=2]           Value for LOW CPU (in cores)
+     * @param  {Number} [options.kubernetes.resources.cpu.micro=0.5]       Value for MICRO CPU (in cores)
+     * @param  {Number} [options.kubernetes.resources.memory.turbo=16]     Value for TURBO memory (in GB)
+     * @param  {Number} [options.kubernetes.resources.memory.high=12]      Value for HIGH memory (in GB)
+     * @param  {Number} [options.kubernetes.resources.memory.low=2]        Value for LOW memory (in GB)
+     * @param  {Number} [options.kubernetes.resources.memory.micro=1]      Value for MICRO memory (in GB)
+     * @param  {String} [options.kubernetes.resources.disk.space]          Value for disk space label (e.g.: screwdriver.cd/disk)
+     * @param  {String} [options.kubernetes.resources.disk.speed]          Value for disk speed label (e.g.: screwdriver.cd/diskSpeed)
+     * @param  {Boolean} [options.kubernetes.dockerFeatureEnabled=false]   Whether to enable docker in docker on the executor k8 container
+     * @param  {Object} [options.kubernetes.nodeSelectors]                 Object representing node label-value pairs
+     * @param  {String} [options.launchVersion=stable]                     Launcher container version to use
+     * @param  {String} [options.prefix='']                                 Prefix for job name
+     * @param  {String} [options.fusebox]                                  Options for the circuit breaker (https://github.com/screwdriver-cd/circuit-fuses)
+     * @param  {Object} [options.requestretry]                             Options for the requestretry (https://github.com/FGRibreau/node-request-retry)
+     * @param  {Number} [options.requestretry.retryDelay]                  Value for retryDelay option of the requestretry
+     * @param  {Number} [options.requestretry.maxAttempts]                 Value for maxAttempts option of the requestretry
+     * @param  {String} [options.ecosystem.cache.strategy]                 Value for build cache - s3, disk
+     * @param  {String} [options.ecosystem.cache.path]                     Value for build cache path if options.cache.strategy is disk
```

Comment:
missing docs for `ecosystem.cache.compress`, `ecosystem.cache.md5check`

Reply:
added